pkg/schedule/coordinator.go

```diff
@@ -52,7 +52,8 @@ const (
 	// pushOperatorTickInterval is the interval try to push the operator.
 	pushOperatorTickInterval = 500 * time.Millisecond
 
-	patrolScanRegionLimit = 128 // It takes about 14 minutes to iterate 1 million regions.
+	// It takes about 1.3 minutes to iterate 1 million regions(with DefaultPatrolRegionInterval=10ms).
```

Comment:
This is an ideal situation, if the consumer side is smaller than the production side, then it will take longer

Reply:
yes

Reply:
Is it better to write `1000000/128*10/60/1000` here to indicate the calculation progress and emphasize it's an ideal situation?